src/react-vite/03-intro-to-jsx/intro-to-jsx.md

```diff
@@ -2,33 +2,46 @@
 @parent learn-react-vite 3
 @outline 3
 
-@description TODO
+@description Learn how to use JSX to define your UI in React.
 
 @body
 
 ## Overview
 
-TODO
+In this section we will:
 
-## Objective 1
+- Learn to use JSX to define our UI
+- Install project assets
+- Create our first component
+- Add variables to JSX
+- Work with loops in JSX
 
-TODO
+## Objective 1 - Creating a UI with JSX
 
 ### Key concepts
 
-TODO
+- What is JSX
+- Loops in JSX
 
-#### Concept 1
+## What is JSX?
```

Comment:
I’m not 100% in love with the placeholder structure I have where these are supposed to go under the “key concepts” header (so this would be an h4) because it does feel like we’re de-emphasizing the concepts. But also, I thought the hierarchy would make sense and be good as soon as we fix the right TOC to be wider. Thoughts?